### PR TITLE
[AMS] Change "region" to "location" in output string: az ams account sp create

### PR DESCRIPTION
**Pull request by hivyas**
**Description**  

Changed the output of the command `az ams account sp create` to say "location" instead of "region" in order to be consistent with our other products.

**Testing Guide**  

`az ams account sp create --account-name [ACCOUNT-NAME] --resource-group [RESOURCE_GROUP]`
**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Review by bennage — APPROVED** [review]
lgtm

**Comment by yonzhan** [comment]
AMS

**Review by xpouyat — COMMENTED** [review]
This may be a breaking change for customers who are using this output in scripts.
Why not keeping also "Region" (plus adding a "Location") ?

File: src/azure-cli/azure/cli/command_modules/ams/operations/sp.py [hunk 1]
```diff
@@ -336,7 +336,7 @@ def _build_sp_result(subscription_id, location, resource_group_name, account_nam
     result = {
         'SubscriptionId': subscription_id,
         'ServicePrincipalName': sp_name,
-        'Region': location,
+        'Location': location,
```
Comment (yungezz) [inline src/azure-cli/azure/cli/command_modules/ams/operations/sp.py:339]: this will be a breaking change to end users

**Comment by yungezz** [comment]
hi @fengzhou-msft could you pls help to follow up this PR? thanks

File: src/azure-cli/azure/cli/command_modules/ams/operations/sp.py (hunk 1 above)
Comment (fengzhou-msft) [reply src/azure-cli/azure/cli/command_modules/ams/operations/sp.py:339]: Can you keep `Region` as well to avoid breaking changes.

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**